projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/secrets/controller/DataJobsSecretsController.java

```diff
@@ -23,19 +23,55 @@
 public class DataJobsSecretsController implements DataJobsSecretsApi {
   static Logger log = LoggerFactory.getLogger(DataJobsSecretsController.class);
 
+  //    private final FeatureFlags featureFlags;
```

Comment:
you can comment all this back in if you follow the suggestion in this comment https://github.com/vmware/versatile-data-kit/pull/2241/files#r1228301649

Reply:
fixed.